.circleci/config.yml

```diff
@@ -90,7 +90,7 @@ jobs:
 
       - run:
           name: set yarn version
-          command: sudo apt-get install yarn=1.6.0-1
+          command: sudo apt-get install yarn=1.16.0
```

Comment:
Should be `1.16.0-1` instead.

(We should probably delete this file, but I can do that in a separate PR)

Reply:
I'll update! What is the `0-1` for?

Reply:
Not sure, but I guess it's how they name versions in apt:

```
circleci@c579d9b13bfa:~/code-dot-org$ apt-cache policy yarn
yarn:
  Installed: 1.16.0-1
  Candidate: 1.16.0-1
  Version table:
 *** 1.16.0-1 0
        500 https://dl.yarnpkg.com/debian/ stable/main amd64 Packages
        100 /var/lib/dpkg/status
     1.15.2-1 0
        500 https://dl.yarnpkg.com/debian/ stable/main amd64 Packages
     1.13.0-1 0
        500 https://dl.yarnpkg.com/debian/ stable/main amd64 Packages
     1.12.3-1 0
        500 https://dl.yarnpkg.com/debian/ stable/main amd64 Packages
     1.12.1-1 0
        500 https://dl.yarnpkg.com/debian/ stable/main amd64 Packages
     1.10.1-1 0
        500 https://dl.yarnpkg.com/debian/ stable/main amd64 Packages
(etc)
```